pkg/infinispan/configuration/server/configuration_test.go

```diff
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"testing"
+
+	"github.com/blang/semver"
+	"github.com/infinispan/infinispan-operator/pkg/infinispan/version"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenerateAndGenerateZeroCapacity(t *testing.T) {
+	baseCfg_expected := "<infinispan\n    xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n    xsi:schemaLocation=\"urn:infinispan:config:15.1 https://infinispan.org/schemas/infinispan-config-15.1.xsd\n                        urn:infinispan:server:15.1 https://infinispan.org/schemas/infinispan-server-15.1.xsd\"\n    xmlns=\"urn:infinispan:config:15.1\"\n    xmlns:server=\"urn:infinispan:server:15.1\">\n\n<cache-container name=\"default\" statistics=\"true\">\n    <global-state unclean-shutdown-action=\"IGNORE\" />\n    \n    <security>\n        <authorization group-only-mapping=\"false\">\n            \n            <cluster-role-mapper />\n            \n            \n        </authorization>\n    </security>\n    \n    <transport cluster=\"${infinispan.cluster.name:}\" node-name=\"${infinispan.node.name:}\" stack=\"image-tcp\"    \n    />\n</cache-container>\n<server xmlns=\"urn:infinispan:server:15.1\">\n    <socket-bindings default-interface=\"public\" port-offset=\"${infinispan.socket.binding.port-offset:0}\">\n        <socket-binding name=\"default\" port=\"${infinispan.bind.port:11222}\"/>\n    </socket-bindings>\n    <security>\n        \n        \n        <security-realms>\n            <security-realm name=\"default\">\n                <server-identities>\n\t\t\t\t\n                </server-identities>\n                \n            </security-realm>\n            \n        </security-realms>\n    </security>\n    <endpoints>\n        <endpoint socket-binding=\"default\" security-realm=\"default\" require-ssl-client-auth=\"true\" />\n    </endpoints>\n</server>\n</infinispan>\n"
```

Comment:
Can we load this from a file?

It's currently almost impossible to read and we will need to update these values in the future when we inevitably have to modify the config.